astroid/exceptions.py

```diff
@@ -100,7 +100,7 @@ class TooManyLevelsError(AstroidImportError):
     def __init__(
         self,
         message="Relative import with too many levels " "({level}) for module {name!r}",
-        **kws
+        **kws,
```

Comment:
Not really sure why the pre-commit changed this?

Reply:
Don't know either. Maybe a small bug with black?
Doesn't really matter though tbh.

Reply:
Black normalize the formatting, if there are multiple line it ends with a comma (and vice versa, add a comma and black will use multiline)

Reply:
Yeah, but nothing changed in the previous commit for `black` to suddenly want to add it (I think).

Reply:
Ho, right. There's a change in the file. though. Maybe due to a new version of black tha thandle kwargs better and apply new formatting only if there is a change in the file ?

Reply:
That is indeed a very good theory!